### PR TITLE
fix: make fullscreen mode trigger automatically

### PR DESCRIPTION
**Pull request by MuhammedKpln**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -34,9 +34,8 @@ FX Client is the first Territorial.io client, offering a better User Interface a
 
 #### The client has a settings menu, from which you can:
 
-13. Make fullscreen mode trigger automatically
```
Comment (peshomir) [inline readme.md:37]: Why is this line removed?

Comment (MuhammedKpln) [reply readme.md:37]: Ah, i misunderstood that.

**Comment by peshomir** [comment]
Thanks for the contribution! I should note that it would be better if formatting changes were done on a separate commit, so the code is easier to review. It's not a big deal though, just a general recommendation.
